students/kangil/westagram/users/views.py

```diff
@@ -39,15 +41,16 @@ def post(self, request):
 class SignInView(View):
     def post(self, request):
         try:
-            
             user_data_input = json.loads(request.body)
             email           = user_data_input['email']
             password        = user_data_input['password']
+            user_data_email = User.objects.get(email = email)
```

Comment:
변수명은 간결하면서도 해당 변수가 참조하고 있는 객체가 어떤 것인지를 명확하게 알 수 있게 만들어주셔야 합니다.
`User.objects.get(email = email)`를 호출하고 반환받는 값은 user 객체이기 때문에 `user`라는 변수명이 적합합니다.
```suggestion
            user= User.objects.get(email = email)
```